translation-web/src/main/java/eu/europeana/api/translation/web/service/TranslationWebService.java

```diff
@@ -44,6 +46,7 @@ public TranslationResponse translate(TranslationRequest translationRequest) thro
       }
       
       try {
+        serviceId = fallback.getServiceId();
```

Comment:
serviceId must be updated only after successfull invocation of the fallback serviceserviceId must be updated only after successfull invocation of the fallback service